src/test/java/tests/PreferencesTests.java

```diff
@@ -1,46 +1,79 @@
 package tests;
 
+import guitests.UITest;
 import org.junit.Test;
-import prefs.GlobalConfig;
+import prefs.SessionConfig;
 import prefs.PanelInfo;
 import prefs.Preferences;
+import ui.TestController;
 
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.List;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.*;
 import static org.mockito.Mockito.*;
 
 public class PreferencesTests {
+
     /**
-     * Tests that Preferences' clearLastOpenBoard method calls GlobalConfig's clearLastOpenBoard once
+     * Tests that Preferences' clearLastOpenBoard method calls SessionConfig's clearLastOpenBoard once
      */
     @Test
     public void testClearLastOpenBoard() {
-        GlobalConfig config = mock(GlobalConfig.class);
-        Preferences prefs = Preferences.load(Preferences.TEST_CONFIG_FILE);
+        SessionConfig sessionConfig = mock(SessionConfig.class);
+        Preferences prefs = TestController.createTestPreferences();
+
+        try {
+            setSessionConfigField(prefs, sessionConfig);
+            prefs.clearLastOpenBoard();
 
-        prefs.global = config;
-        prefs.clearLastOpenBoard();
+        } catch (IllegalAccessException | NoSuchFieldException e) {
```

Comment:
I was given this suggestion before. Don't catch it; make the method throws this exception, so JUnit will fail if these exception occurs. The test name also needs to be updated to follow new convention. `prefClearLastOpenBoard_mockSessionConfig_sessionConfigRespectiveMethodCalled`